### PR TITLE
Remove redundancy from actor page, refs #12718

### PR DESCRIPTION
**Pull request by mcantelon**
On the left sidebar of the actor detail page, descriptions related by events were showing in the "subject of" section in addition to in the event type's section. Removed display of these events in the "subject of" section.

### CODE REVIEW

File: apps/qubit/modules/actor/actions/relatedInformationObjectsAction.class.php [hunk 1]
```diff
@@ -90,18 +90,13 @@ public static function getRelatedInformationObjects($actorId, $page, $limit, $ev
...
-
-            // Use nested query and mapping object to allow querying
-            // over the actor and event ids from the same event
-            $queryNested = new \Elastica\Query\Nested();
-            $queryNested->setPath('dates');
-            $queryNested->setQuery($queryBoolDates);
 
+            foreach (QubitTerm::getEventTypes() as $eventType) {
+                $queryNested = self::nestedActorAndEventTypeQuery($actorId, $eventType->id);
+                $queryBool->addMustNot($queryNested);
```
Comment (djjuhasz) [inline apps/qubit/modules/actor/actions/relatedInformationObjectsAction.class.php:96]: Is it necessary to explicitly exclude the event type relationships with a "MustNot" query?  I would expect simply not adding the events (like before this change) to the query to be sufficient. 😕

**Review by djjuhasz — APPROVED** [review]
I have one question about the functionality @mcantelon, but other than that the PR looks good to me.

File: apps/qubit/modules/actor/actions/relatedInformationObjectsAction.class.php (hunk 1 above)
Comment (mcantelon) [reply apps/qubit/modules/actor/actions/relatedInformationObjectsAction.class.php:96]: It's a bit convoluted. ;)

For issue 12718 it was requested to add the display of non-creator actors (created via the RAD template) in the name access points section of the description's page and to make them searchable, as name access points, in the advanced search. Commit 8ac0c7551a9e771df5ccba30ceca2c4429aa0efc (December of 2019) implemented this by indexing non-creator actors as name access points in the ElasticSearch index.

The only problem with this, though, is that the left sidebar of the actor page now ends up displaying these actors twice: once as a name access point and once grouped by its event type. This PR fixes this display issue by suppressing the display of them in the name access point section of the sidebar.

Comment (djjuhasz) [reply apps/qubit/modules/actor/actions/relatedInformationObjectsAction.class.php:96]: Ah, thanks for the explanation.  Sounds like this is the best solution for the current search data model.

**Pull request merged**